### ISSUE TITLE
New release 2.0.0?

### ISSUE THREAD

**Issue by wesleybl** [collective/plone.app.transmogrifier#26]
I made changes to the package to work with Python 3 and Plone 5.2. I performed a migration from a portal to Plone 5.2 and Python 3.8.11 with the master branch and it worked correctly.

Could someone please create a 2.0.0 release? I don't have permission on the pypi.

@tisto @lrowe @gotcha @agitator @hvelarde @mjpieters

**Comment by tisto** [comment]
@wesleybl this package is missing the required infrastructure to do a release. Seems I don't have pypi permissions to grant you access. Someone should give you permission to make a release and also add "collective", and "plone" (and maybe "tisto").

**Comment by mauritsvanrees** [comment]
> Can you (or someone) please add me in
> 
> `Restrict who can dismiss pull request reviews`
> 
> from protection of master branch?

Done, and also for "Restrict who can push to matching branches". For the last one it did not work in [collective.transmogrify](https://github.com/collective/collective.transmogrifier/issues/28), but maybe the combination helps.

Alternatively, create a release and push it to a separate branch in a PR and then it can be merged.
But you need to look on PyPI first and see who can grant you release rights.

Honestly, we could also remove all (or most) branch protection rules. It is not like this is a central super important package that gets multiple commits each day and where people tend to push mistakes directly to master.

**Comment by thet** [comment]
I added @tisto, @mauritsvanrees, plone and collective as owner of https://pypi.org/project/plone.app.transmogrifier/

**Comment by wesleybl** [comment]
@mauritsvanrees 

> Done, and also for "Restrict who can push to matching branches". For the last one it did not work in collective.transmogrify, but maybe the combination helps.

Unfortunately it didn't help. See the description of "Restrict who can push to matching branches":

```
Specify people, teams or apps allowed to push to matching branches. Required status checks will still prevent these people, teams and apps from merging if the checks fail. 
```

So I think as while as it's checked "Require status checks to pass before merging" only users of repository with write permission (or maybe admins) can make push to master.

> I added @tisto, @mauritsvanrees, plone and collective as owner of https://pypi.org/project/plone.app.transmogrifier/

@mauritsvanrees @tisto can you make a new release? Please merge #27 first.

**Comment by mauritsvanrees** [comment]
I have released 2.0.0.
